src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -244,6 +245,7 @@ const formConfig = {
                   primaryPhone: {
```

Comment:
:thinking: @markgreenburg Do you remember why we weren't using the phone number definition from `us-forms-system`?

Reply:
I don't actually remember...I think we switched to using the `usaPhone` definition in all-claims and using `phoneUI()`. That might be a good idea here as well 🤷‍♂️